package.json

```diff
@@ -73,9 +73,11 @@
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.1.1",
+    "fast-check": "^3.14.0",
     "jasmine": "^4.6.0",
     "json-schema-to-typescript": "^13.1.1",
     "minimist": "^1.2.8",
+    "parse-address": "^1.1.2",
```

Comment:
This looks pretty unmaintained but given the size probably acceptable risk.

Reply:
yeah agreed - I spoke about this at the time. The idea was that it would come up in code review and we'd decide to either keep a dependency, or vendor it in etc.

Reply:
Maybe it shouldn't be a dev dep though as we're integrating it into the generated code right?

Reply:
Ahh yeah that's a good point.

Reply:
done.